metricbeat/helper/prometheus/prometheus.go

```diff
@@ -60,14 +71,20 @@ type httpfetcher interface {
 
 // NewPrometheusClient creates new prometheus helper
 func NewPrometheusClient(base mb.BaseMetricSet) (Prometheus, error) {
+	cfg := defaultConfig()
+	err := base.Module().UnpackConfig(cfg)
+	if err != nil {
+		return nil, err
+	}
+
 	http, err := helper.NewHTTP(base)
```

Comment:
Sorry, I have only thought about this now, but, would it make sense to add this to the HTTP helper so this can be leveraged in other modules too? This could be done as a future improvement in any case.

Reply:
This is a good point!